code/game/objects/items/devices/radio/headset.dm

```diff
@@ -632,6 +632,16 @@
 	. = ..()
 	spy_bug.nametag = "CL Radio"
 
+/obj/item/device/radio/headset/almayer/mcl/sec
+	name = "corporate security radio headset"
+	spy_bug_type = null
```

Comment:
this being null is whats causing the map tests to fail i believe
[18:37:11]RUNTIME: Cannot modify null.nametag. - code/game/objects/items/devices/radio/headset.dm@633